### DISCUSSION TITLE
Issue with readme

### DISCUSSION THREAD

**Discussion by Blake777999** [flybywiresim/aircraft#4776] [Help] [answered]
under downloads it says "Warning: The development (master) and experimental versions have been forked as a separate package/aircraft in MSFS, and will appear separate from the default A320neo. Existing Asobo A320neo liveries will be incompatible with the new package." Even though the stable aircraft is a different aircraft now and not a part of the native A320

**Comment by scoopscode** [comment, marked as answer]
> under downloads it says "Warning: The development (master) and experimental versions have been forked as a separate package/aircraft in MSFS, and will appear separate from the default A320neo. Existing Asobo A320neo liveries will be incompatible with the new package." Even though the stable aircraft is a different aircraft now and not a part of the native A320

Thanks for the heads up! Will fix soon!
